build.gradle

```diff
@@ -64,6 +64,9 @@ subprojects {
   apply plugin: 'maven' // make pom files for deployment
   apply plugin: 'nebula.maven-base-publish'
 
+  compileJava {options.encoding = "UTF-8"}
```

Comment:
These blocks should use newlines and match the formatting of the rest of the file.

Reply:
I see, updated in latest commit